docs/user_guide/README.md

```diff
@@ -51,17 +51,48 @@ the generated project and use the normal QGIS editing tools.
     * `Orphan metadata` displays tables not existing in the database itself, but the metadata exists in the
     `dataset` table.
     * `Orphan tables` is the reverse. It displays tables existing in the database but in the `dataset` table.
-1. On the `Dataset` table, open the attribute table, switch on **Edition** mode and add a new row.
+
+### Editing metadata
+
+Before creating metadata for our layer/table, you need to implement some table. To edit this table you need to 
```

Comment:
> Before creating metadata for **our** layer/table, **you** need to implement some table.

It looks strange, you are using different subjects within the same sentence.

In French : Avant de créer la métadonnée pour **notre** table, **vous** devez...
Il faut être cohérent avec les sujets